packages/templates/wasm/golang/go.mod

```diff
@@ -2,6 +2,6 @@ module example.com/template-wasm-go
 
 go 1.18
 
-require github.com/polywrap/go-wrap wrap-0.1
+require github.com/polywrap/go-wrap v0.0.0-20230712212127-6895977d63c2
```

Comment:
please leave as `wrap-0.1`

Reply:
It automatically changes to the commit version when we run `go mod tidy` but I will update it.

Reply:
I think We should use tag for the releases

Reply:
agreed